homeassistant/components/todo/__init__.py

```diff
@@ -106,7 +106,7 @@ def _validate_supported_features(
         if desc.service_field not in call_data:
             continue
         if not supported_features or not supported_features & desc.required_feature:
-            raise ValueError(
+            raise ServiceValidationError(
```

Comment:
It would be great if we make the errors translatable too.
https://developers.home-assistant.io/docs/internationalization/core/#exceptions

Do you think we can add it to this PR?

Reply:
Sure, i'll have a look.